### PR TITLE
feat: OPX-3.2.0

### PR DESCRIPTION
**Pull request by Tejaswi-Goel**
* Feature: Subinterface creation enable for Virtual Machine
* Update: Deleting FEC attribute for disabled FEC
* Update: Modified the log message for successful deferred interface "enable"
* Update: New vlan attribute to set dynamic arp filter for vlan
* Update: Add vxlan Lock during remote endpoint addition/deletion to avoid race condition

Signed-off-by: Tejaswi Goel <Tejaswi_Goel@Dell.com>

### CODE REVIEW

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now 2479d78dbf433464a84d1a9bbdeb0c65fff66874

File: src/stats/nas_stats_virtual_ntwrk.cpp [hunk 1]
```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright (c) 2018 Dell Inc.
```
Comment (rakeshdatta) [inline src/stats/nas_stats_virtual_ntwrk.cpp]: Change the Copyright

File: src/unit_test/test_nas_get_interface.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2018 Dell Inc.
```
Comment (rakeshdatta) [inline src/unit_test/test_nas_get_interface.py]: change the copyright

File: src/vrf/nas_vrf_api.cpp [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018 Dell Inc.
+ * Copyright (c) 2017 Dell Inc.
```
Comment (rakeshdatta) [inline src/vrf/nas_vrf_api.cpp]: change the copyright

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now bdf0da16b8a48bc5f90cfbb77c00934c44745be9

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now 90573864b51b442bfdcf4261981c8aff1e90ac71

**Pull request merged**